doxygen/dox/sw_changes_1.10.dox

```diff
@@ -0,0 +1,660 @@
+    to change the applied filters. This version of h5repack understands the file free space feature
+    and handles the file and metadata appropriately.
+
+<h4>Virtual Dataset or VDS:</h4>
+\li #H5Pset_virtual and h5pset_virtual_f - Sets the mapping between virtual and source datasets.
+\li #H5Pget_virtual_count and h5pget_virtual_count_f - Retrieves the number of mappings for the
+    virtual dataset.
+\li #H5Pget_virtual_vspace and h5pget_virtual_vspace_f - Retrieves a dataspace identifier for the
+    selection within the virtual dataset used in the mapping.
+\li #H5Pget_virtual_srcspace and h5pget_virtual_srcspace_f - Retrieves a dataspace identifier
+    for the selection within the source dataset used in the mapping.
+\li #H5Pget_virtual_dsetname and h5pget_virtual_dsetname_f - Retrieves the name of a source
+    dataset used in the mapping.
+\li #H5Pget_virtual_filename and h5pget_virtual_filename_f - Retrieves the filename of a source
+    dataset used in the mapping.
+\li #H5Pset_virtual_printf_gap and h5pset_virtual_printf_gap_f - Sets maximum number of missing
+    source files and/or datasets with printf-style names when getting the extent of an unlimited virtual dataset.
+\li #H5Pget_virtual_printf_gap and h5pget_virtual_printf_gap_f - Returns maximum number of missing
+    source files and/or datasets with printf-style names when getting the extent for an unlimited virtual dataset.
+\li #H5Pset_virtual_view and h5pset_virtual_view_f - Sets the view of the virtual dataset to include
+    or exclude missing mapped elements.
+\li #H5Pget_virtual_view and h5pget_virtual_view_f - Retrieves the view of a virtual dataset.
+
+<h4>Supporting Functions:</h4>
+\li #H5Sis_regular_hyperslab and h5sis_regular_hyperslab_f - Determines whether a hyperslab selection is regular.
+\li #H5Sget_regular_hyperslab and h5sget_regular_hyperslab_f - Retrieves a regular hyperslab selection.
+
+<h4>Modified Functions:</h4>
+The following pre-exising functions have been modified to understand virtual datasets.
+\li #H5Pset_layout and h5pset_layout_f - Specifies the layout to be used for a dataset.
+    Virtual dataset, #H5D_VIRTUAL, has been added to the list of layouts available through this function.
+\li #H5Pget_layout and h5pget_layout_f - Retrieves the layout in use for a dataset.
+    Virtual dataset, #H5D_VIRTUAL, has been added to the list of layouts.
+
+<h4>Partial Edge Chunks:</h4>
+\li #H5Pset_chunk_opts - Sets a partial edge chunk option in a dataset access property list.
+\li #H5Pget_chunk_opts - Retrieves partial edge chunk option setting from a dataset access property list.
+
+<h4>Relative Pathnames for External Links:</h4>
+\li #H5Pset_elink_prefix - Sets the prefix to be applied to external link paths. 
+\li #H5Pget_elink_prefix - Retrieves the prefix applied to external link paths. 
+
+These functions enable the user to specify the locations of external files.
+(These functions are not yet documented.)
+
+H5Pget_elink_prefix 
```

Comment:
This one has a link, so line 525 should be removed.

Reply:
No that line probably should just add a "as of version x.y.z)